e2e/__tests__/retainAllFiles.test.ts

```diff
@@ -0,0 +1,25 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import runJest from '../runJest';
+
+describe('Retain All Files Integration', () => {
+  test('valid test within node_modules', () => {
+    const config = {
+      haste: {
+        retainAllFiles: true,
+      },
+    };
+
+    const {exitCode} = runJest('retain-all-files', [
+      '--config',
+      JSON.stringify(config),
+      'retainAllFiles.test.js',
+    ]);
```

Comment:
```suggestion
    const {exitCode} = runJest('retain-all-files');
```

shouldn't be necessary to pass args